### PR TITLE
Dataset fleet

### PR DESCRIPTION
**Pull request by xjqbest**
(1) add fleet save model
(2) add fleet load model

### CODE REVIEW

File: paddle/fluid/framework/fleet/fleet_wrapper.h [hunk 1]
```diff
@@ -128,6 +128,10 @@ class FleetWrapper {
   // create client to client connection
   void CreateClient2ClientConnection();
 
+  // param = 0, save all feature
```
Comment (jacquesqiao) [inline paddle/fluid/framework/fleet/fleet_wrapper.h]: param  ---> mode?

File: paddle/fluid/framework/fleet/fleet_wrapper.cc [hunk 2]
```diff
@@ -352,6 +352,29 @@ int FleetWrapper::RegisterClientToClientMsgHandler(int msg_type,
   return 0;
 }
 
+void FleetWrapper::LoadModel(const std::string& path,
+                             const std::string& mode) {
+  auto ret = pslib_ptr_->_worker_ptr->load(path, mode);
```
Comment (guru4elephant) [inline paddle/fluid/framework/fleet/fleet_wrapper.cc:357]: you should add pslib macro here

Comment (guru4elephant) [reply paddle/fluid/framework/fleet/fleet_wrapper.cc:357]: same as below

File: paddle/fluid/framework/fleet/fleet_wrapper.h [hunk 3]
```diff
@@ -128,6 +128,13 @@ class FleetWrapper {
   // create client to client connection
   void CreateClient2ClientConnection();
 
+  // mode = 0, load all feature
+  // mode = 1, laod delta feature
+  void LoadModel(const std::string& path, const std::string& mode);
```
Comment (guru4elephant) [inline paddle/fluid/framework/fleet/fleet_wrapper.h:133]: why not use int mode?

File: paddle/fluid/framework/fleet/fleet_wrapper.h [hunk 4]
```diff
@@ -128,6 +128,13 @@ class FleetWrapper {
...
+  // mode = 0, save all feature
+  // mode = 1, save delta feature
+  void SaveModel(const std::string& path, const std::string& mode);
```
Comment (guru4elephant) [inline paddle/fluid/framework/fleet/fleet_wrapper.h:136]: why not use int mode?

File: python/paddle/fluid/incubate/fleet/parameter_server/__init__.py [hunk 5]
```diff
@@ -227,19 +227,49 @@ def is_server(self):
...
+            raise ValueError("mode must be 0 or 1, but mode=%s" % mode)
+        mode = str(mode)
+        self._role_maker._barrier_worker()
         if self._role_maker._is_first_worker():
-            self._fleet_ptr.init_model()
+            self._fleet_ptr.load_model(load_path, mode)
         self._role_maker._barrier_worker()
 
-    def save_pserver_model(self, save_path):
+    def save_pserver_model(self, save_path, mode=0):
```
Comment (guru4elephant) [inline python/paddle/fluid/incubate/fleet/parameter_server/__init__.py:252]: the argument of mode needs very detailed comments to clarify the difference between save all and save delta

**Pull request closed without merging**